diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 6ebafc0acd445e8ba4b6de859ebfdd27aeb32944..1c1495a4646d43effe59a9ab33690db776282692 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2010,6 +2010,7 @@ jumbo_split_static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_posix || is_fuchsia) {
