diff --git a/chrome/browser/profiles/profile_manager.cc b/chrome/browser/profiles/profile_manager.cc
index 16c500023e3f84554dc28d81ef31be1cee25f868..d39a5c92c5674ba06605f4201682f1576ef646c1 100644
--- a/chrome/browser/profiles/profile_manager.cc
+++ b/chrome/browser/profiles/profile_manager.cc
@@ -1492,9 +1492,11 @@ void ProfileManager::EnsureActiveProfileExistsBeforeDeletion(
   const base::FilePath last_used_profile_path =
       GetLastUsedProfileDir(user_data_dir_);
   const base::FilePath guest_profile_path = GetGuestProfilePath();
+  const base::FilePath tor_profile_path = BraveProfileManager::GetTorProfilePath();
   Profile* last_used_profile = GetProfileByPath(last_used_profile_path);
   if (last_used_profile_path != profile_dir &&
       last_used_profile_path != guest_profile_path &&
+      last_used_profile_path != tor_profile_path &&
       last_used_profile != nullptr &&
       !last_used_profile->IsLegacySupervised()) {
     FinishDeletingProfile(profile_dir, last_used_profile_path);
@@ -1507,6 +1509,7 @@ void ProfileManager::EnsureActiveProfileExistsBeforeDeletion(
     base::FilePath cur_path = profile->GetPath();
     if (cur_path != profile_dir &&
         cur_path != guest_profile_path &&
+        cur_path != tor_profile_path &&
         !profile->IsLegacySupervised() &&
         !IsProfileDirectoryMarkedForDeletion(cur_path)) {
       OnNewActiveProfileLoaded(profile_dir, cur_path, std::move(callback),
@@ -1526,6 +1529,7 @@ void ProfileManager::EnsureActiveProfileExistsBeforeDeletion(
     // legacy-supervised.
     if (cur_path != profile_dir &&
         cur_path != guest_profile_path &&
+        cur_path != tor_profile_path &&
         !entry->IsLegacySupervised() &&
         !IsProfileDirectoryMarkedForDeletion(cur_path)) {
       fallback_profile_path = cur_path;
@@ -1884,6 +1888,7 @@ void ProfileManager::ScheduleForcedEphemeralProfileForDeletion(
     base::FilePath entry_path = entry->GetPath();
     if (entry_path == profile_dir ||
         entry_path == GetGuestProfilePath() ||
+        entry_path == BraveProfileManager::GetTorProfilePath() ||
         entry->IsLegacySupervised() ||
         IsProfileDirectoryMarkedForDeletion(entry_path))
       continue;
