diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 694796bf8dbba514ae9f09e51186e53be14ae50e..b27f0a7c649ae0ce2909be3839dfeabb5b82ea76 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -1467,6 +1467,13 @@ class CONTENT_EXPORT ContentBrowserClient {
   // to native code, and as such whether its log messages should be recorded.
   virtual bool IsBuiltinComponent(BrowserContext* browser_context,
                                   const url::Origin& origin);
+
+  // Brave-speicific: allows the embedder to modify the referrer string
+  // according to user preferences.
+  virtual void MaybeHideReferrer(BrowserContext* browser_context,
+                                 const GURL& request_url,
+                                 const GURL& document_url,
+                                 Referrer* referrer) {}
 };
 
 }  // namespace content
